.github/workflows/deploy.yaml

```
@@ -45,13 +45,46 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_KEY }}
           aws-region: ${{ vars.AWS_REGION }}
 
+      - name: 현재 서비스 상태 확인
+        run: |
+          # 현재 desired count만 확인
+          CURRENT_COUNT=$(aws ecs describe-services \
+            --cluster ${{ vars.ECS_CLUSTER }} \
+            --services ${{ vars.ECS_SERVICE }} \
+            --query 'services[0].desiredCount' \
+            --output text)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)